README.md

```diff
@@ -1,33 +1,31 @@
 # RQT2
 
-## Install ROS2
+## Setup
 
-In order to reproduce our setup follow the instructions on the ros2 wiki for [installing ROS2 from source on Ubuntu 18.04](https://index.ros.org/doc/ros2/Linux-Development-Setup/).
+In order to reproduce our setup follow the instructions on the ros2 wiki for [installing ROS2 from source on Ubuntu 18.04](https://index.ros.org/doc/ros2/Linux-Development-Setup/) stopping before the section: *Get ROS 2.0 code*.
 
 TODO: After Crystal release, support building RQT2 on top of debian ROS2 packages
 
 ## Build From Source
 
-### Install Dependencies
-
-This section is only necessary if you haven't install ROS2 from source:
```

Comment:
Make it clear that source install = building RQT *and* ROS2 from source. A mixed approach (ROS2 from debian) is not currently documented (which makes sense given the Crystal status).